controller-manager/src/main/resources/templates/ServiceMonitor-console.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: enmasse-console
+  namespace: {{ index .Params "Namespace" }}
+  labels:
+    monitoring-key: middleware
+    app: enmasse
+spec:
+  selector:
+    matchLabels:
+      app: enmasse
+      component: console
+  endpoints:
+  - port: metrics
+    path: /metrics
+    metricRelabelings:
+      - sourceLabels: [__name__]
+        targetLabel: __name__
+        replacement: "enmasse_${1}"
+  namespaceSelector:
+    matchLabels:
+      monitoring-key: middleware
```

Comment:
@lulf @robshelly what is the role of the monitoring-key?  Do I need to attach this to the console service as I see done for the address-space-controller.

Reply:
I think it might be necessary for it to be picked up by the application monitoring operator, but I'm not sure if it is actually necessary or not anymore.

Reply:
It's still necessary as far as I know

Reply:
We agreed offline it wasn't necessary.